src/processScript/index.ts

```diff
@@ -260,7 +265,7 @@ export async function processScript(code: string, {
 			{
 				name: `hackmud-script-manager`,
 				async transform(code, id) {
-					if (id.startsWith(`/`) && !id.includes(`/node_modules/`))
+					if (isPath(id) && !id.includes(`${pathSeparator}node_modules${pathSeparator}`))
```

Comment:
I think you can get rid of `isPath()` and instead use [isAbsolute()](https://nodejs.org/api/path.html#path_path_isabsolute_path).

Rest looks good though, ty!